lib/Model/Type.cpp

```diff
@@ -403,7 +403,7 @@ isValidPrimitiveSize(PrimitiveTypeKind::Values PrimKind, uint8_t BS) {
     return BS == 1 or BS == 2 or BS == 4 or BS == 8 or BS == 16;
 
   case PrimitiveTypeKind::Float:
-    return BS == 2 or BS == 4 or BS == 8 or BS == 12 or BS == 16;
+    return BS == 2 or BS == 4 or BS == 8 or BS == 10 or BS == 12 or BS == 16;
```

Comment:
I kinda got lost on this, did you manage to have a `static_assert(sizeof(some float) == 10)`?
Can you godbolt it?

Reply:
I wasn't able to make a simple case for this. I have the PDB file that contains the type only. I will try further.

Reply:
I was not able to produce the test. Anyhow, we faced it in the ```PDB``` that was created by ```VS```... I have no idea how it is being produced -- seems like it represents a library that may be written in another programming language.

Reply:
Supporting something we're not able to reproduce saddens me.
Let's either ignore that float upon import, or at least put a comment above this part stating why we want to support `10`.

Reply:
Yeah... I will try to make a comment for it.